src/ui/components/Logs/Logs.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react'
+
+const basePath = document.head.querySelector('base')?.getAttribute('href') || ''
+
+export const Logs = ({
+  queueName,
+  jobId,
+}: {
+  queueName: string
+  jobId: string | number | undefined
+}) => {
+  const [state, setState] = useState([])
+
+  useEffect(() => {
```

Comment:
Components should not know how to make AJAX requests, it breaks the entire encapsulation.
All "actions" are under `actions` object (which get passed already to JobCard).

This way you don't need to pass the queue name every where.

Pls refactor this 🙏🏼